src/main/java/dev/j3fftw/litexpansion/machine/MetalForge.java

```diff
@@ -0,0 +1,115 @@
+package dev.j3fftw.litexpansion.machine;
+
+import dev.j3fftw.litexpansion.Items;
+import dev.j3fftw.litexpansion.LiteXpansion;
+import io.github.thebusybiscuit.slimefun4.core.multiblocks.MultiBlockMachine;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.utils.SlimefunUtils;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.api.Slimefun;
+import me.mrCookieSlime.Slimefun.cscorelib2.inventory.InvUtils;
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.Sound;
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockFace;
+import org.bukkit.block.Dispenser;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MetalForge extends MultiBlockMachine {
+
+    public static final RecipeType METAL_FORGE = new RecipeType(
```

Comment:
Rename this constant to just `RECIPE_TYPE`

```suggestion
    public static final RecipeType RECIPE_TYPE = new RecipeType(
```

Reply:
okay will also do that for Refined smeltery

Reply:
then it doesnt seem to work